<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLineEdit" name="inputBox7">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="inputBox6">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="inputBox4">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="inputBox9">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="inputBox3">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="inputBox5">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="inputBox8">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="inputBox2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="inputBox1">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLineEdit" name="resultBox1">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="resultBox8">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="resultBox2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="resultBox7">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="resultBox4">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="resultBox6">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="resultBox3">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="resultBox9">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="resultBox5">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="2">
      <widget class="QLineEdit" name="inputBox6_2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="inputBox4_2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="inputBox2_2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="inputBox5_2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="inputBox9_2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="inputBox7_2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="inputBox3_2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="inputBox1_2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="inputBox8_2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>203</width>
      <height>103</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="1">
      <widget class="QPushButton" name="echButton1">
       <property name="text">
        <string>기약행사다리꼴</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="powerButton1">
       <property name="text">
        <string>n제곱</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="trButton1">
       <property name="text">
        <string>전치행렬</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="inverseButton1">
       <property name="text">
        <string>역행렬</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="adjButton1">
       <property name="text">
        <string>수반행렬</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="clearButton1">
       <property name="text">
        <string>초기화</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>203</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="2" column="0">
      <widget class="QPushButton" name="echButton2">
       <property name="text">
        <string>기약행사다리꼴</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="adjButton2">
       <property name="text">
        <string>수반행렬</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="inverseButton2">
       <property name="text">
        <string>역행렬</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="trButton2">
       <property name="text">
        <string>전치행렬</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="powerButton2">
       <property name="text">
        <string>n제곱</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="clearButton2">
       <property name="text">
        <string>초기화</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>82</width>
      <height>103</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="plusButton">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="minusButton">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mulButton">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="powerInput1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>n</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="powerInput2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>230</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>n</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>96</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>FuncMode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="insertLeft">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>insert L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="insertRight">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>390</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>insert R</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>203</width>
      <height>103</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="1">
      <widget class="QPushButton" name="clearButton1_2">
       <property name="text">
        <string>해시안</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="gradButton1">
       <property name="text">
        <string>그래디언트</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="powerButton1_2">
       <property name="text">
        <string>자코비안</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="inputBox1_4">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>90</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>866</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menucalculator">
    <property name="title">
     <string>calculator</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionmatrix"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuteah">
    <property name="title">
     <string>teah</string>
    </property>
    <addaction name="actionasdfadfs"/>
   </widget>
   <addaction name="menucalculator"/>
   <addaction name="menuteah"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionmatrix">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>matrix</string>
   </property>
  </action>
  <action name="actionasdfadfs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>asdfadfs</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
